shells/pipes-shell/device.js

```diff
@@ -72,6 +72,20 @@ const shellApi = {
     log('flushing caches');
     Utils.env.loader.flushCaches();
     marshalRecipeContext();
+  },
+  registerMessageHandler(handler) {
```

Comment:
We've shifted Android to build on pipes-shell-2, should `pipes-shell` be deprecated and we should make the needed changes in this PR there instead? Or is `pipes-shell` still a thing and we need to evolve it? @yuangu-google

Reply:
We don't use pipes-shell in Android any more. Only pipes-shell-2 is used.